HW9.md

```diff
@@ -0,0 +1,130 @@
+# Homework
+
+### 1. add secondary ip address to you second network interface enp0s8. Each point must be presented with commands and showing that new address was applied to the interface. To repeat adding address for points 2 and 3 address must be deleted (please add deleting address to you homework log) Methods:
+
+   #### 1.1 using ip utility (stateless)
+
+```bash
+[alia2@localhost ~]$ sudo ip addr add 192.168.122.197/24 dev enp0s8
+[alia2@localhost ~]$  ip addr show enp0s8
+3: enp0s8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
+    link/ether 08:00:27:b2:db:28 brd ff:ff:ff:ff:ff:ff
+    inet 172.20.10.7/28 brd 172.20.10.15 scope global noprefixroute dynamic enp0s8
+       valid_lft 85976sec preferred_lft 85976sec
+    inet 192.168.122.197/24 scope global enp0s8
+       valid_lft forever preferred_lft forever
+    inet6 fe80::1dfc:3a01:b4e4:5d6f/64 scope link noprefixroute
+       valid_lft forever preferred_lft forever
+[alia2@localhost ~]$ sudo ip addr del 192.168.122.197/24 dev enp0s8
+[alia2@localhost ~]$  ip addr show enp0s8
+3: enp0s8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
+    link/ether 08:00:27:b2:db:28 brd ff:ff:ff:ff:ff:ff
+    inet 172.20.10.7/28 brd 172.20.10.15 scope global noprefixroute dynamic enp0s8
+       valid_lft 85925sec preferred_lft 85925sec
+    inet6 fe80::1dfc:3a01:b4e4:5d6f/64 scope link noprefixroute
+       valid_lft forever preferred_lft forever
+```
+
+   #### 1.2 using centos network configuration file (statefull)
+
+```bash
+[alia2@localhost ~]$ sudo nano /etc/sysconfig/network-scripts/ifcfg-enp0s8:1
+DEVICE=enp0s8:1
+Type=Ethernet
+ONBOOT=yes
+NM_CONTROLLED=no
+BOOTPROTO=none
+IPADDR=192.168.122.197
+PREFIX=24
+[alia2@localhost ~]$ reboot
+[alia2@localhost ~]$ ip addr show enp0s8
+3: enp0s8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
+    link/ether 08:00:27:b2:db:28 brd ff:ff:ff:ff:ff:ff
+    inet 192.168.122.197/24 brd 192.168.122.255 scope global noprefixroute enp0s8
+       valid_lft forever preferred_lft forever
+    inet 172.20.10.7/28 brd 172.20.10.15 scope global noprefixroute dynamic enp0s8
+       valid_lft 86283sec preferred_lft 86283sec
+    inet6 fe80::1dfc:3a01:b4e4:5d6f/64 scope link noprefixroute
+       valid_lft forever preferred_lft forever
+[alia2@localhost ~]$ sudo ip addr del 192.168.122.197/24 dev enp0s8
```

Comment:
Why did you add IP address in one way, but delete it in another? 
Will that action remove the newly created configuration file?